packages/neuron-ui/src/exceptions/index.ts

```diff
@@ -110,3 +110,19 @@ export class ValueReservedException extends Error {
     this.i18n = { value }
   }
 }
+
+export type ErrorWithI18n = {
+  message: string
+  code: ErrorCode
+  i18n?: Record<string, string>
+}
+
+export function isErrorWithI18n(error: any): error is ErrorWithI18n {
+  return (
+    typeof error === 'object' &&
+    error !== null &&
```

Comment:
`typeof error === 'object' && error !== null` is redundant because `error?.message` covers them.